DEVELOPMENT.md

```diff
@@ -196,5 +196,6 @@ to start a debugger to which attach from your IDE.
    ```
    expose the `KIND_CLUSTER_NAME` env variable with the appropriate name of the kind cluster:
    ```bash
-   export KIND_CLUSTER_NAME=$(kubectl config get-clusters | grep -P '(?<=kind-).*' -o)
+   export KIND_CLUSTER_NAME=$(kind -q get clusters)
```

Comment:
How is this related to getting the kind from the resource name?

Reply:
It is not, I just added the change because I faced problems when testing in my local with the other command.

It is ok for you to place it here or should I create separate commit in the PR? or a separate PR for this line?

I thought that with a separate PR it will be a little overkill as I guess I should create an issue for it.

Reply:
I'm OK with it, but I've seen on other projects that people complain when things are mixed.  I think this is mainly for reverting.  

If you do make additional changes, I tend to just note them in the description so that the reviewer is aware.